### PR TITLE
dev/core#711 remove Google+

### PR DESCRIPTION
**Pull request by agileware-pengyi**
Overview
----------------------------------------
I am trying to push this forward, so here is PR for removing Google+ icon from the social network template.

Before
----------------------------------------
Google+ reference included

After
----------------------------------------
Google+ reference removed

Comments
----------------------------------------

Agileware ref: CIVICRM-1243

### CODE REVIEW

**Comment by yashodha** [comment]
@agileware-pengyi Is that all the references in the codebase?

**Comment by seamuslee001** [comment]
Looks like there are references to it in `templates/CRM/Contribute/Form/ContributionPage/Settings.hlp` `xml/templates/civicrm_data.tpl`  `CRM/Dedupe/Merger.php` as well there is a reference in `CRM/Upgrade/Incremental/sql/4.5.alpha1.mysql.tpl` but that should stay IMO

**Comment by agileware-pengyi** [comment]
@seamuslee001 @yashodha I've updated the branch.

**Comment by yashodha** [comment]
@agileware-pengyi  thanks!

File: xml/templates/civicrm_data.tpl [hunk 1]
```diff
@@ -740,7 +740,6 @@ VALUES
    (@option_group_id_website, 'Work',     1, 'Work',     NULL, 0, 1, 1, NULL, 0, 0, 1, NULL, NULL, NULL),
    (@option_group_id_website, 'Main',     2, 'Main',     NULL, 0, NULL, 2, NULL, 0, 0, 1, NULL, NULL, NULL),
    (@option_group_id_website, 'Facebook', 3, 'Facebook', NULL, 0, NULL, 3, NULL, 0, 0, 1, NULL, NULL, NULL),
-   (@option_group_id_website, 'Google+',  4, 'Google_',  NULL, 0, NULL, 4, NULL, 0, 0, 1, NULL, NULL, NULL),
```
Comment (colemanw) [inline xml/templates/civicrm_data.tpl]: Upgrade SQL needed to remove this.

**Comment by eileenmcnaughton** [comment]
Well google says it's so.... https://timesofindia.indiatimes.com/gadgets-news/google-consumer-accounts-to-be-discontinued-on-april-2-heres-all-you-need-to-know/articleshow/67768585.cms?from=mdr

**Comment by eileenmcnaughton** [comment]
Well google says it's so.... https://timesofindia.indiatimes.com/gadgets-news/google-consumer-accounts-to-be-discontinued-on-april-2-heres-all-you-need-to-know/articleshow/67768585.cms?from=mdr

**Comment by eileenmcnaughton** [comment]
I'm going to merge this - the upgrade PR can follow but I don't think it blocks

**Pull request merged**